packages/install-ses/install-ses.js

```diff
@@ -18,3 +18,8 @@ lockdown({ errorTaming: 'unsafe' });
 // Even on non-v8, we tame the start compartment's Error constructor so
 // this assignment is not rejected, even if it does nothing.
 Error.stackTraceLimit = Infinity;
+
+harden(TextEncoder.prototype);
+harden(TextEncoder);
+harden(TextDecoder.prototype);
+harden(TextDecoder);
```

Comment:
Even with today's too-complicated `harden` lines 22 and 24 above are redundant. `TextEncoder.prototype` is reachable by own property traversal from `TextEncoder` and so will be included in the set hardened when starting with `TextEncoder`.